app/config/github.yml

```diff
@@ -23,12 +28,14 @@ parameters:
                 - app.subscriber.status_change_by_comment_subscriber
                 - app.subscriber.needs_review_new_pr_subscriber
                 - app.subscriber.bug_label_new_issue_subscriber
+                - app.subscriber.status_change_on_push_subscriber
             # secret: change_me
         symfony/symfony-docs:
             subscribers:
                 - app.subscriber.status_change_by_comment_subscriber
                 - app.subscriber.needs_review_new_pr_subscriber
                 - app.subscriber.bug_label_new_issue_subscriber
+                - app.subscriber.status_change_on_push_subscriber
 
         # used in a functional test
         weaverryan/symfony:
```

Comment:
Very friendly ping @weaverryan (also, any news on enabling carson for the docs?)